cli/endpoints/online/triton/single-model/single-triton-lite.ipynb

```diff
@@ -0,0 +1,291 @@
+      "Requirement already satisfied: cffi>=1.12 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from cryptography>=2.1.4->azure-storage-blob) (1.14.5)\n",
+      "Requirement already satisfied: pycparser in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=2.1.4->azure-storage-blob) (2.20)\n",
+      "Requirement already satisfied: isodate>=0.6.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (0.6.0)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (2020.12.5)\n",
+      "Requirement already satisfied: requests-oauthlib>=0.5.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from msrest>=0.6.18->azure-storage-blob) (1.3.0)\n",
+      "Requirement already satisfied: idna<3,>=2.5 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (2.10)\n",
+      "Requirement already satisfied: chardet<5,>=3.0.2 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (4.0.0)\n",
+      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.10.0->azure-storage-blob) (1.26.4)\n",
+      "Requirement already satisfied: oauthlib>=3.0.0 in /home/arzooaneja/anaconda3/envs/azmlenv/lib/python3.7/site-packages (from requests-oauthlib>=0.5.0->msrest>=0.6.18->azure-storage-blob) (3.1.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install azure-storage-blob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "successfully downloaded model: densenet_onnx\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "from pathlib import Path\n",
+    "from src.model_utils import download_triton_models, delete_triton_models\n",
+    "\n",
+    "prefix = Path(\".\")\n",
+    "download_triton_models(prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Register model\n",
+    "\n",
+    "The registered model should follow the Triton specified model folder structure for Triton Inference Server to be able to load it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subscripton = \"13e50845-67bc-4ac5-94db-48d493a6d9e8\"\n",
+    "resource_group = \"chrjia-rg\"\n",
+    "workspace = \"chrjia-eastus2euap2\"\n",
```

Comment:
Remove actual subscription GUIDs, resource group names.  This is a general comment for all recipes.

Reply:
Issue Resolved